tests/ui/associated-types/hr-associated-type-projection-1.stderr

```diff
@@ -16,7 +16,7 @@ LL | trait UnsafeCopy<'a, T: Copy>
 LL | where
 LL |     for<'b> <Self as UnsafeCopy<'b, T>>::Item: std::ops::Deref<Target = T>,
    |                                                                ^^^^^^^^^^ required by this bound in `UnsafeCopy`
-help: consider further restricting this bound
+help: consider further restricting this bound with  `<Target = T>`
```

Comment:
nit: It looks like an extra space snuck in here.

Reply:
That output is already not great... I think I'll do a iollow up just reverting to the previous message for that one case. There's another for "with `, Output = Ty`" which I don't love either.